docs/source/common_issues.rst

```diff
@@ -808,3 +808,34 @@ not necessary:
     class NarrowerArgument(A):
         def test(self, t: List[int]) -> Sequence[str]:  # type: ignore[override]
             ...
+
+Unpacking Dictionary in keyword arguments
+-----------------------------------
+If you are unpacking a dictionary into a Callable's arguments, then
+you need to ensure the types of dictionary members by providing either
+a TypedDict or a variable with type Dict[str, Any].
+
+For example, the following function calls are valid:
```

Comment:
Just a suggestion - maybe an example that shows some offending cases or invalid function calls would help drive the point home? (otherwise good job!)

Reply:
Sure, will add them.